src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -901,7 +901,18 @@ private boolean placeBlock(@NotNull final BlockPos pos, final Block block, @NotN
             }
         }
 
-        if (block instanceof BlockDoor)
+        if (block instanceof BlockBeetroot)
+        {
+            //We need to do the beetroot first as it use a different property but BlockBeetroot instance of BlockCrops
+            //we don't want fully grown plants
+            world.setBlockState(pos,blockState.withProperty(BlockBeetroot.BEETROOT_AGE,0),0x03);
```

Comment:
Not out of my head. You can find out by going through the call path of that method.